Documentation/doxygen/IDE.md

```diff
@@ -0,0 +1,39 @@
+IDEs for development
+====================
+
+Clion
+-----
+
+We highly recommend to use `Clion` for your development. It has a robust support for C/C++ languages and `CMake`.
+
+
+Eclipse for Fortran
+-------------------
+
+Unfortunately, Clion does not support Fortran well. Therefore, we recommend to have an installation for eclipse parallel edition which comes with a Photran plugin.
```

Comment:
Photran plugin. typo?

Reply:
No. It is correct. See https://www.eclipse.org/photran/

Reply:
ok